src/PointwiseFunctions/AnalyticSolutions/Elasticity/HalfSpaceMirror.hpp

```diff
@@ -100,18 +100,18 @@ class HalfSpaceMirror {
     static constexpr Options::String help{
         "Workspace size for numerical integrals. Increase if integrals fail to "
         "reach the prescribed tolerance at large distances relative to the "
-        "beam width. The default values for workspace size and tolerances "
+        "beam width. The suggested values for workspace size and tolerances "
         "should accommodate distances of up to ~100 beam widths."};
     static type lower_bound() noexcept { return 1; }
-    static type default_value() noexcept { return 350; }
+    static type suggested_value() noexcept { return 350; }
   };
```

Comment:
Changing the default_value to suggested_value makes me a bit queasy as I feel a warning discourages the user from increase the workspace size. Such a change is unavoidable though when increasing the extents of the domain/`BeamWidth`.

Reply:
I think this concern would be resolved by toning down the "warning" wording, as discussed above (https://github.com/sxs-collaboration/spectre/pull/2522#issuecomment-703176493)